pints/tests/test_toy_eight_schools_logpdf.py

```diff
@@ -31,6 +31,15 @@ def test_basic(self):
         val = f([1, 0.5, 0.4, 1, 1, 1, 1, 1, 1, 1])
         self.assertAlmostEqual(val, -38.24061255483484, places=6)
 
+        # Default settings with non-ones input
+        f1, dp = f.evaluateS1(list(n + 1 for n in range(10)))
```

Comment:
usually you'd just write `evaluateS1([n + 1 for ...])`

Reply:
Fixed in commit f866c8f